### DISCUSSION TITLE
t2 Нижегородская область конфиг

### DISCUSSION THREAD

**Discussion by Yany1944** [dovecoteescapee/ByeDPIAndroid#165] [General]
Нужна помощь, перестало всё работать (

**Comment by 15zemlekop** [comment]
![Screenshot_2024-10-09-15-19-54-95_2b145053e9452336c3f6817dabd56d09](https://github.com/user-attachments/assets/a4b75f09-3db7-4885-93fd-770db00d625a)
![Screenshot_2024-10-09-15-19-57-26_2b145053e9452336c3f6817dabd56d09](https://github.com/user-attachments/assets/598976a6-95fd-4118-bc5a-f57fe451f971)
